lib/src/resource_owner_password_grant.dart

```diff
@@ -32,6 +32,9 @@ import 'utils.dart';
 /// The scope strings will be separated by the provided [delimiter]. This
 /// defaults to `" "`, the OAuth2 standard, but some APIs (such as Facebook's)
 /// use non-standard delimiters.
+///
+/// [getParameters] can be a function used to parse parameters out of responses from hosts that do not
+/// respond with application/json or application/x-www-form-urlencoded bodies.
```

Comment:
This documentation needs to be more thorough. Explain what arguments are passed to this callback. Explain that the spec requires a JSON response, and that the return value should be structured like that response. Link to the spec so people know what structure to parse the response into.